### PR TITLE
pgAdmin4 with configurable background

### PR DESCRIPTION
**Pull request by thedetukov**
Implement image pgAdmin4 with configurable background

### CODE REVIEW

File: .github/workflows/docker-image-release.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # See https://github.com/docker/build-push-action#readme
+      # See https://github.com/docker/build-push-action/blob/b1aeb1103e6b3b5648dbd6deaf0559919456ca6b/docs/advanced/tags-labels.md
+      - name: Build And Push
+        id: docker_build
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          platforms: linux/arm64,linux/amd64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6
```
Comment (theanurin) [inline .github/workflows/docker-image-release.yml]: Looks like you need to remove

- linux/ppc64le
- linux/s390x
- linux/386
- linux/arm/v7
- linux/arm/v6

File: .github/workflows/docker-image-snapshot-tag.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # See https://github.com/docker/build-push-action#readme
+      # See https://github.com/docker/build-push-action/blob/b1aeb1103e6b3b5648dbd6deaf0559919456ca6b/docs/advanced/tags-labels.md
+      - name: Build And Push
+        id: docker_build
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          platforms: linux/arm64,linux/amd64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6
```
Comment (theanurin) [inline .github/workflows/docker-image-snapshot-tag.yml]: Looks like you need to remove

- linux/ppc64le
- linux/s390x
- linux/386
- linux/arm/v7
- linux/arm/v6

File: .github/workflows/docker-image-release.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update image README and description
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          repository: ${{ secrets.DOCKERHUB_USERNAME }}/pgadmin4
+          short-description: "Apache pgadmin4 is a software versioning and revision control system distributed as open source under the Apache License."
```
Comment (theanurin) [inline .github/workflows/docker-image-release.yml]: Incorrect `short-description`

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "thedetukov"
```
Comment (theanurin) [inline .vscode/settings.json]: Pls replace `thedetukov` -> `theanurin`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+[![Docker Image Version](https://img.shields.io/docker/v/thedetukov/pgadmin4?sort=date&label=Version)](https://hub.docker.com/r/thedetukov/pgadmin4/tags)
```
Comment (theanurin) [inline README.md]: Pls replace `thedetukov` -> `theanurin`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+# Spec
+
+## Environment variables
+
+No any variables
+
+## Expose ports
+
+* `tcp/39000` - pgadmin4 listening endpoint
```
Comment (theanurin) [inline README.md]: Pls replace `tcp/39000` -> `tcp/80`

File: docker/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,9 @@
+FROM dpage/pgadmin4@sha256:9450d844f6d70951c873fad60c8c420cf1f3f87cf7abd8be7f8136c88ca75ebb
+
+USER root
+
+RUN apk add --no-cache sudo
+
+COPY docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint-pgadmin4.sh
+
```
Comment (theanurin) [inline docker/Dockerfile:12]: Pls add
```
ENV REDIS_COMMANDER_BG_COLOR=
```
See more https://docs.docker.com/engine/reference/builder/#env

File: README.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+## Volumes
+
+* `/docker-images` - Root of the pgAdmin repositories
+
+# Inside
+
+* dpage/pgadmin4:latest
```
Comment (theanurin) [inline README.md]: Here we have to define pgAdmin version (not image)

File: README.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+No any variables
+
+## Expose ports
+
+* `tcp/39000` - pgadmin4 listening endpoint
+
+## Volumes
+
+* `/docker-images` - Root of the pgAdmin repositories
```
Comment (theanurin) [inline README.md]: Pls write

No any volumes

File: README.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
+
+# Spec
+
+## Environment variables
+
+No any variables
```
Comment (theanurin) [inline README.md]: Pls add definition of `REDIS_COMMANDER_BG_COLOR`

File: README.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+[![Docker Pulls](https://img.shields.io/docker/pulls/thedetukov/pgadmin4?label=Pulls)](https://hub.docker.com/r/thedetukov/pgadmin4)
+[![Docker Stars](https://img.shields.io/docker/stars/thedetukov/pgadmin4?label=Docker%20Stars)](https://hub.docker.com/r/thedetukov/pgadmin4)
+
+# pgAdmin
+
+[pgAdmin](https://www.pgadmin.org/) is the most popular and feature rich Open Source administration and development platform for PostgreSQL, the most advanced Open Source database in the world.
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
```
Comment (theanurin) [inline README.md]: Describe real reason of the image.

**Review by theanurin — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 4 above)
Comment (thedetukov) [reply .vscode/settings.json]: Fixed

File: .github/workflows/docker-image-release.yml (hunk 3 above)
Comment (thedetukov) [reply .github/workflows/docker-image-release.yml]: Fixed

File: .vscode/settings.json (hunk 4 above)
Comment (thedetukov) [reply .vscode/settings.json]: Fixed

File: .github/workflows/docker-image-snapshot-tag.yml (hunk 2 above)
Comment (thedetukov) [reply .github/workflows/docker-image-snapshot-tag.yml]: Removed unused OS.

File: .github/workflows/docker-image-release.yml (hunk 1 above)
Comment (thedetukov) [reply .github/workflows/docker-image-release.yml]: Fixed. Removed unused OS.

Comment (thedetukov) [reply .github/workflows/docker-image-release.yml]: image

Comment (thedetukov) [reply .github/workflows/docker-image-release.yml]: Fixed.

File: .github/workflows/docker-image-snapshot-tag.yml (hunk 2 above)
Comment (thedetukov) [reply .github/workflows/docker-image-snapshot-tag.yml]: uses: docker/build-push-action@v4
        with:
          push: true
          platforms: linux/arm64,linux/amd64
          file: docker/Dockerfile

Comment (thedetukov) [reply .github/workflows/docker-image-snapshot-tag.yml]: with:
          push: true
          platforms: linux/arm64,linux/amd64

File: README.md (hunk 5 above)
Comment (thedetukov) [reply README.md]: Fixed.

File: .github/workflows/docker-image-release.yml (hunk 1 above)
Comment (thedetukov) [reply .github/workflows/docker-image-release.yml]: Fixed. Removed unused OS.

File: docker/Dockerfile (hunk 7 above)
Comment (thedetukov) [reply docker/Dockerfile:12]: Added ENV PGADMIN_BG_COLOR=

File: README.md (hunk 9 above)
Comment (thedetukov) [reply README.md]: Fixed.

File: README.md (hunk 10 above)
Comment (thedetukov) [reply README.md]: Fixed.

File: README.md (hunk 11 above)
Comment (thedetukov) [reply README.md]: Fixed.

File: README.md (hunk 6 above)
Comment (thedetukov) [reply README.md]: Changed tcp/39000 -> tcp/80

File: README.md (hunk 8 above)
Comment (thedetukov) [reply README.md]: Added version to pgAdmin4.

File: .github/workflows/docker-image-release.yml [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update image README and description
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          repository: ${{ secrets.DOCKERHUB_USERNAME }}/pgadmin4
+          short-description: "Pgadmin4 the most advanced Open Source database."
```
Comment (theanurin) [inline .github/workflows/docker-image-release.yml]: Wrong description

**Review by theanurin — APPROVED** [review]

**Pull request merged**